src/main/java/emissary/output/filter/AbstractFilter.java

```diff
@@ -169,12 +173,53 @@ protected void initializeOutputTypes(@Nullable final Configurator config) {
             this.outputTypes = config.findEntriesAsSet("OUTPUT_TYPE");
             this.logger.debug("Loaded {} output types for filter {}", this.outputTypes.size(), this.outputTypes);
             this.denylist = config.findEntriesAsSet("DENYLIST");
+            this.validateDenylist();
+            this.wildCardDenylist = this.denylist.stream()
+                    .filter(i -> i.endsWith("*"))
+                    .map(i -> i.substring(0, i.length() - 1))
+                    .collect(Collectors.toSet());
+            this.wildCardDenylist.forEach(i -> this.denylist.remove(i + "*"));
             this.logger.debug("Loaded {} ignorelist types for filter {}", this.denylist.size(), this.denylist);
+            this.logger.debug("Loaded {} wildcard suffix ignorelist types for filter {}", this.wildCardDenylist.size(), this.wildCardDenylist);
         } else {
             this.logger.debug("InitializeCustom has null filter config");
         }
     }
 
+    protected void validateDenylist() {
+        Pattern charSet = Pattern.compile("^[\\w*]+[\\w*.]*[\\w*]+$"); // Match if acceptable characters are in correct order
+        Pattern repeatedPeriods = Pattern.compile("\\.\\."); // Match if any sequential `.` characters
+        Pattern typeWildcardFormat = Pattern.compile("^(\\*|\\w+)$"); // Match if String is `*` or word sequence
+        Pattern viewWildcardFormat = Pattern.compile("^[\\w.]*\\*?$"); // Match if String is word sequence with optional `*` suffix
+        final String errorPrefix = "Invalid filter configuration: `DENYLIST = %s` ";
+
+        for (String entry : this.denylist) {
+            if (charSet.matcher(entry).matches() && !repeatedPeriods.matcher(entry).matches()) {
+                String[] names = entry.split("\\.", 2);
+                String filetype = names[0];
+                if (!typeWildcardFormat.matcher(filetype).matches()) {
+                    throw new EmissaryRuntimeException(String.format(errorPrefix +
+                            "filetype `%s` must be wildcard `*` only or sequence of [A-Z, a-z, 0-9, _].",
+                            entry, filetype));
```

Comment:
I see what you did with extracting the similar errorPrefix but I think it is a little cleaner to just have it part of each exception message. Open to discuss it more.

```suggestion
                    throw new EmissaryRuntimeException(String.format("Invalid filter configuration: `DENYLIST = %s`filetype `%s` must be wildcard `*` only or sequence of [A-Z, a-z, 0-9, _].",
                            entry, filetype));
```

Reply:
That's fair. I'm fine with removing the errorPrefix variable and just writing it out. It is a long message though and I think it would be good to keep it on several lines.